biofefi/options/file_paths.py

```diff
@@ -14,6 +14,20 @@ def uploaded_file_path(file_name: str, experiment_path: Path) -> Path:
     return experiment_path / file_name
 
 
+def raw_data_path(file_name: str, experiment_path: Path) -> Path:
+    """Create the full path to the directory to save raw data.
+
+    Args:
```

Comment:
add `file_name` to the docstring please. Also create a test please. The other functions in this model have tests too. Check out `tests/options/test_file_paths.py`